eventing/triggers/new-trigger-filters.adoc

```diff
@@ -0,0 +1,47 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="new-trigger-filters"]
+= New trigger filters
```

Comment:
I'd call this "Advanced trigger filters"

Reply:
like that suggestion

Reply:
Changed the name